### ISSUE TITLE
Add Versioning to this repo

### ISSUE THREAD

**Issue by morriswinkler-simplesurance** [knq/xo#107]
Hello, 

We are having hard time using this tool.

The mist recent updates just broke our project because of the changes you made to the uuid and time.Time types. Last working rev was  1f80f9b7376a92a5e10d7dcebefd4a5c7c6ce0e0.

Could you be so polite and tag that rev so we can get it with gopkg.in

**Comment by nocive** [comment]
bump

**Comment by kenshaw** [comment]
Yes, xo has almost finished a complete "end-to-end" rewrite. I haven't had time to quite finish it, but when it is done, it will be versioned properly, as well as have vendored dependencies added. I would like to say this will be done immediately, but it's based on when I am able to find enough free time to finish it up.

**Comment by nocive** [comment]
Thanks for the reply!
Good to know that this isn't forgotten and that will be coming some time soon :)

**Comment by morriswinkler-simplesurance** [comment]
👍

**Comment by rohmanhm** [comment]
Seems @kenshaw adding go modules support at
https://github.com/xo/xo/commit/d69d59ce52a9e2006eac2c042bca217a202aa003

According to https://github.com/golang/go/wiki/Modules you need to push a new tag to make it works with modules.

```
Your new module release becomes available to consumers once you push the new release tags.
```
